src/components/appointmentForm/AppointmentForm.js

```diff
@@ -1,67 +1,70 @@
-import React, { useState, useEffect } from 'react';
+import React from "react";
 
-import { ContactPicker } from '../contactPicker/ContactPicker';
-import DatePicker from "react-datepicker";
+import { ContactPicker } from "../contactPicker/ContactPicker";
 
 export const AppointmentForm = ({
   contacts,
-  addAppointment
+  setTitle,
+  setContact,
+  setDate,
+  setTime,
+  handleSubmit,
 }) => {
-    const initContact = (contacts.length > 0) ? contacts[0].name : '';
-    const [title, setTitle] = useState('');
-    const [contact, setContact] = useState(initContact);
-    const [date, setDate] = useState(new Date());
-    const [submitValid, setSubmitValid] = useState(false);
+  const today = new Date();
+  const month = (today.getMonth() + 1 < 10 ? "0" : "") + (today.getMonth() + 1);
```

Comment:
There's a "cleaner" way of doing this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#to_get_date_month_and_year_or_time

Reply:
The date input "min" format needs 2 digit month and day, plus month is 0 to 11. I like this way right now since I can test the number values. I'll look at making it slicker down the line.